README.md

```diff
@@ -2,18 +2,100 @@
 
 The auctioneer bot monitors a Blend pool to create and bid on auctions. This includes user liquidation auctions, bad debt auctions, and interest auctions. The auctioneer focuses on completeness and pool safety over profit, but the existing code can be modified to meet any use case.
 
+For more information please see [the Blend docs](https://docs.blend.capital/tech-docs/core-contracts/lending-pool).
+
+## Safety
+
+This software is made available under the MIT License, which disclaims all warranties in relation to the project and which limits the liability of those that contribute and maintain the project, including Script3. You acknowledge that you are solely responsible for any use of this software and you assume all risks associated with any such use.
+
 ## Getting Started
 
-The auctioneer is packaged into a single docker container. To build the image, run:
+It is recommended to run the auctioneer bot as a docker container. The image can be pulled with:
+
+```bash
+docker pull script3/auctioneer-bot:latest
+```
+
+To run the container, you will need to provide a volume mount from the host to `/app/data` with a config file named `config.json` at the root of the volume. Please see [Configuration](#configuration) for documentation of the configuration file.
```

Comment:
I think this is a tad confusing. I would suggest "To run the container, you will need to mount a directory from the host to '/app/data'. The directory must contain a 'config.json' file at the root of the directory. .....